csvcubed/csvcubed/cli/inspect/inspectsparqlmanager.py

```diff
@@ -196,11 +220,97 @@ def select_dsd_code_list_and_cols(
 
     Member of :file:`./inspectsparqlmanager.py`
 
-    :return: `CodelistInfoSparqlResult``
+    :return: `CodelistInfoSparqlResult`
     """
     results: List[ResultRow] = select(
-        _get_query_string_from_file(SPARQLQueryFileName.SELECT_CODE_LISTS_AND_COLS),
+        _get_query_string_from_file(SPARQLQueryFileName.SELECT_CODELISTS_AND_COLS),
         rdf_graph,
         init_bindings={"dsd_uri": URIRef(dsd_uri)},
     )
     return map_codelists_sparql_result(results, json_path)
+
+
+def select_csvw_table_schema_file_dependencies(
+    rdf_graph: Graph,
+) -> CSVWTableSchemaFileDependenciesResult:
+    """
+    Queries the table schemas of the given csvw json-ld.
+
+    Member of :file:`./inspectsparqlmanager.py`
+
+    :return: `CSVWTabelSchemasResult`
+    """
+    results: List[ResultRow] = select(
+        _get_query_string_from_file(
+            SPARQLQueryFileName.SELECT_CSVW_TABLE_SCHEMA_FILE_DEPENDENCIES
+        ),
+        rdf_graph,
+    )
+
+    # TODO: Need to map from relative paths to absolute paths here.
+    return map_csvw_table_schemas_result(results)
+
+
+def select_qb_dataset_url(rdf_graph: Graph, dataset_uri: str) -> DatasetURLResult:
+    """
+    Queries the url of the given qb:dataset.
+
+    Member of :file:`./inspectsparqlmanager.py`
+
+    :return: `DatasetURLResult`
+    """
+    if not dataset_uri.startswith("file://"):
+        raise InvalidCsvFilePathException
```

Comment:
Strictly speaking I wouldn't say this is true. Do you recall why you added this bit of code?